templates/careers/administration.html

```diff
@@ -15,7 +15,7 @@ <h1 class="p-heading--2"><strong>Administration</strong></h1>
     </div>
     <div class="col-6">
       <p class="u-sv2">Canonical is a completely new kind of organisation. Almost entirely distributed, we are a global team of technology leaders who collaborate online to enable the transformation of enterprise software to open source. We have a few office locations, but almost everybody works from home. Teams get together every few months at locations around the world to plan and coordinate their projects.</p>
-      <a class="p-button--positive" href="/careers/all">See all roles</a>
+      <a class="p-button--positive" href="/careers/all?filter=Administration">See all Administration roles</a>
```

Comment:
@marinacastejon do you not think this damages the user experience of visiting a careers department page? I would have thought the very next thing I would want to do is view the open roles for that department.

Reply:
@anthonydillon @mtruj013 The behaviour I proposed (checked and agreed with other UX colleagues) is that:
- When there are open roles in that department, the button in the department page says "See all [department name] roles". On click it takes users to the Open roles page with the filters turned on for that department. 
- When there aren't any open roles in that department, the button says "See all roles", on click it takes you to the open roles page showing all roles, no filters applied. Otherwise, the user would go straight to an empty open roles page. As discussed with the UX team and MariaPaula, we will also in another issue, add a sentence telling users that there are no open roles in that department next to the CTA green button. 

Regarding the copy doc, I am not sure how you usually do it when the button label changes dynamically, but perhaps the copy doc should have the standard label "See all [department name] roles". It doesn't see an easy way to maintain the copy docs to update them every time a department goes from having some roles to no roles? 
Let me know what you think.

Reply:
Ok I'll do that for all department pages. Therefore as a follow up to this branch.